marklogic-data-hub/src/test/java/com/marklogic/hub/flow/connected/RunConnectedStepsViaMlcpTest.java

```diff
@@ -49,7 +49,8 @@ void ingestAndMapWithOptions() {
         verifyMappedDataInFinal(recordCount);
     }
 
-    @Test
+    // SJS transform doesn't report back errors. See https://github.com/marklogic/marklogic-contentpump/issues/180
+    //@Test
```

Comment:
Do we have a wait to know/remember to bring back this tests once that is resolved?